### DISCUSSION TITLE
Wind compass

### DISCUSSION THREAD

**Discussion by Loreexe** [PranshulGG/WeatherMaster#158] [Ideas]
Sync the wind direction with the real-time compass on your phone so that when you turn your phone the arrow points in the same direction as the wind

**Comment by PranshulGG** [comment]
> Sync the wind direction with the real-time compass on your phone so that when you turn your phone the arrow points in the same direction as the wind

I did try to implement it, but it didn’t work. It’s not an ignored feature—I’ll have to give it another shot.
